src/main/java/oop/Customer.java

```diff
@@ -0,0 +1,45 @@
+package oop;
+
+import java.util.Scanner;
+
+public class Customer {
+    private String name;
+    private int age;
+
+    public Customer(String name, int age) {
+        this.name = name;
+        this.age = age;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    public int getAge() {
+        return age;
+    }
+
+    public void setAge(int age) {
+        this.age = age;
+    }
+
+    Scanner scan = new Scanner(System.in);
+
+
+    public Object selectHall(){
+        System.out.println("Please select a hall: IMAX or REAL2D");
+        String movie = scan.nextLine().toUpperCase();
+
+
+        if(movie.matches("IMAX")){
+            return new Imax().showListOfFilms();
+        }else if(movie.matches("REAL2D")){
+            return new Real2D().showListOfFilms();
+        }
+        return null;
```

Comment:
it is a bad practice to return null from methods as it can cause NullPointerException later
if you need to return list of films, it is better to use Collections.emptyList() or smth like this